lib/dataduck/table.rb

```diff
@@ -157,7 +165,7 @@ def extract_query(source_spec, destination = nil)
 
     def extract_by_clause(value)
       if value
-        "WHERE #{ self.extract_by_column } >= '#{ value }'"
+        "WHERE #{ self.extract_by_column } > '#{ value }'"
```

Comment:
The >= was intentional. In very rare cases, with just >, you might miss a row if they have the exact same value. Values for the extract_by_column should be guaranteed to be nondecreasing but not necessarily strictly increasing. For instance, consider the following:

During load 1:

created_at, name
2015-05-01 10:00:00, Jeff

During load 2:
created_at, name
2015-05-01 10:00:00, Jeff
2015-05-01 10:00:00, Ian